compiler/codegen/OMRCodeGenerator.hpp

```diff
@@ -616,8 +619,8 @@ class OMR_EXTENSIBLE CodeGenerator
    bool isMaterialized(TR::Node *); // no virt, cast
    bool isMaterialized(int64_t); // no virt, cast
    bool materializesLargeConstants() { return false;}
-   int32_t getLargestNegConstThatMustBeMaterialized() {return -32769;} // no virt, cast
-   int32_t getSmallestPosConstThatMustBeMaterialized() {return 32768;} // no virt, cast
+   int64_t getLargestNegConstThatMustBeMaterialized() {return -32769;}
+   int64_t getSmallestPosConstThatMustBeMaterialized() {return 32768;}
```

Comment:
These are not arbitrary. The two default values here are max/min 16-bit signed numbers used by PPC. On Z, I have override functions that return max/min 32-bit signed numbers.

I'll leave a comment on these two.